Apps/Admin/Common/Models/UserFeedbackView.cs

```diff
@@ -83,9 +84,15 @@ public UserFeedbackView(IList<UserFeedbackTagView> tags)
     public string Email { get; set; } = string.Empty;
 
     /// <summary>
-    /// Gets the feedback admin tags.
+    /// Gets or sets a value indicating whether the feedback has been modified or not.
     /// </summary>
-    public IList<UserFeedbackTagView> Tags { get; } = new List<UserFeedbackTagView>();
+    public bool IsDirty { get; set; }
```

Comment:
This property is only relevant for the front-end and should not be included in this model.

To track this in the front-end, you can create an extended model - see ExtendedDelegateInfo as the best example of this, which uses an AutoMapper profile for easy conversion.  The conversion happens in the effect method.

Reply:
Changed.